linked_list.rb

```diff
@@ -48,18 +59,45 @@ def length
   # method to return the value of the nth element from the beginning
   # assume indexing starts at 0 while counting to n
   def find_nth_from_beginning(n)
-    puts "Not implemented"
+    index = 0
+    current = @head
+    if current
+      while index < n

```

Comment:
What if there are less than n+1 number of nodes in the list?
To fix for that scenario, Reverse the order of if and while statements and if current become null sooner than the n, return null.